frontend/src/components/Hacknight.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-container fluid text-xs-center>
+    <v-row align="center" justify="center">
+      <v-col class="d-flex" cols="12" sm="6">
+        <v-alert
+          type="error"
+          :value="getError ? true : false"
+          transition="slide-y-transition"
+          dismissible
+          >{{ getError }}</v-alert
+        >
+      </v-col>
+    </v-row>
+    <v-row align="center" justify="center">
+      <v-col class="d-flex" cols="12" sm="6">
+        <v-select
+          v-model="selectedHacknight"
+          :items="getHacknights"
+          item-text="date"
+          item-value="id"
+          label="Select Hacknight"
+          @input="onGetHacknight"
+        >
+          <template v-slot:append-outer>
+            <v-btn
+              color="blue-grey"
+              class="ma-2 white--text"
+              @click="onCreateHacknight"
+              style="top: -12px"
```

Comment:
should we add style like here in template? It mixes concerns a bit. Maybe we could move it to css?